### PR TITLE
Fix bug in bleve's zap stored command, formatting

### PR DESCRIPTION
**Pull request by abhinavdangeti**
+ Handle the _id field special case
+ Format the output for better readability

### CODE REVIEW

File: cmd/bleve/cmd/zap/dict.go [hunk 1]
```diff
@@ -45,8 +45,8 @@ var dictCmd = &cobra.Command{
 		vellumLen, read := binary.Uvarint(data[addr : addr+binary.MaxVarintLen64])
 		fmt.Printf("vellum length: %d\n", vellumLen)
 		fstBytes := data[addr+uint64(read) : addr+uint64(read)+vellumLen]
-		fmt.Printf("raw vellum data % x\n", fstBytes)
-		fmt.Printf("dictionary:\n\n")
+		fmt.Printf("raw vellum data:\n % x\n", fstBytes)
```
Comment (steveyen) [inline cmd/bleve/cmd/zap/dict.go]: this was there before, but feels like an extra space in the "% x" is wrong?

File: cmd/bleve/cmd/zap/explore.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ var exploreCmd = &cobra.Command{
 		vellumLen, read := binary.Uvarint(data[addr : addr+binary.MaxVarintLen64])
 		fmt.Printf("vellum length: %d\n", vellumLen)
 		fstBytes := data[addr+uint64(read) : addr+uint64(read)+vellumLen]
-		fmt.Printf("raw vellum data % x\n", fstBytes)
+		fmt.Printf("raw vellum data:\n % x\n", fstBytes)
```
Comment (steveyen) [inline cmd/bleve/cmd/zap/explore.go]: (looks like same here)

**Comment by mschoch** [comment]
% x is by design to put spaces between each byte value, not a typo

**Comment by steveyen** [comment]
> % x is by design to put spaces between each byte value, not a typo

cool - learned something new.

looks good to me

**Pull request merged**